### PR TITLE
Section restructuring

### PR DESCRIPTION
**Pull request by jlevy**
Based on discussions w @RacheltheEditor and other reviewers.

- Moving equity comp basics up.
- Consolidating IPOs and sales under fundamentals of stock corps
- Moving selling of private stock down to evaluating

Adjusting basics definitions to use fewer later concepts and add a few forward references.

Very little edits or new material except the very beginning of equity comp basics.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -230,7 +230,11 @@ it would be misleading to explain one without the other.
 We cover material in logical order, so that if you do read the earlier sections first,
 later sections on the interactions of tax and compensation will be clearer.
 
-The first sections are about how companies work and how they grow:
+We start with [**Equity Compensation Basics**](#equity-compensation-basics): What compensation
+and equity compensation are.
+
+But before we get much further, we need to focus on how companies work and how they grow.
+These are important concepts to learn in order to understand what equity might be worth.
```
Comment (RacheltheEditor) [inline README.md]: 👍

File: README.md [hunk 2]
```diff
@@ -440,6 +495,69 @@ Many decisions around granting equity to employees are approved by the board of
...
+It’s also called a **sale** of the acquired company.
+
+🄳 A sale or IPO is often called an **exit** or **liquidity event**. Sales, dissolutions, and
+bankruptcy are all called **liquidations**.
+
+🄳 The ability to buy and sell stock is called **liquidity**. In startups and many private
+companies, it is often hard to sell stock until the company is sold or IPOs, so there is
+little or no liquidity for shareholders until those events occur.
+
+Often people wish they could sell stock in a private company, because they would prefer
```
Comment (jlevy) [inline README.md]: bit of new content here

File: README.md [hunk 3]
```diff
@@ -310,10 +311,78 @@ This Guide *complements but does not replace* the advice you get from profession
 Working with the support of a professional can help you make better decisions when you
 have an understanding of the topic yourself, and know what questions to ask.
 
-## Fundamentals of Stock Corporations
+## Equity Compensation Basics
+
+### Compensation and equity
+
+🄳 **Compensation** is what you get in exchange for working for a company, including cash
```
Comment (jlevy) [inline README.md]: reworked

File: README.md [hunk 4]
```diff
@@ -230,28 +230,28 @@ it would be misleading to explain one without the other.
 We cover material in logical order, so that if you do read the earlier sections first,
 later sections on the interactions of tax and compensation will be clearer.
 
-The first sections are about how companies work and how they grow:
+We start with [**Equity Compensation Basics**](#equity-compensation-basics): What compensation
+and equity compensation are.
```
Comment (RacheltheEditor) [inline README.md]: "What compensation and equity are, and why equity is used by some companies as compensation."

File: README.md [hunk 5]
```diff
@@ -230,28 +230,28 @@ it would be misleading to explain one without the other.
...
 
-- [**Fundamentals of Stock Corporations**](#fundamentals-of-stock-corporations): To understand
-  exactly what equity is, it’s essential to know how companies organize their ownership, how
-  stock works, and how ownership typically evolves.
-- [**Startups and Growth**](#startups-and-growth): While not everyone reading this works at an early
-  stage company, those who do can benefit from understanding the particulars of how these
-  companies raise money and grow.
+[**Fundamentals of Stock Corporations**](#fundamentals-of-stock-corporations) covers how companies
+organize their ownership, how stock is issued, public and private companies, and IPOs and
+liquidity.
```
Comment (RacheltheEditor) [inline README.md]: Consider, "public and private companies, and how the fate of private companies affects equity." (Could put "IPOs and liquidity" as a parenthetical there. Just establish the relationship b/n each of these here to smooth out the logical narrative.)

File: README.md [hunk 6]
```diff
@@ -230,28 +230,28 @@ it would be misleading to explain one without the other.
...
 
-Then compensation:
+While not everyone reading this works at an early stage company, those who do can benefit
+from understanding the particulars of [**Startups and Growth**](#startups-and-growth).
```
Comment (RacheltheEditor) [inline README.md]: We could say something (either here or in the intro the section itself) about how 'even if you work for a public company now, the private origins of public companies might be of interest to you, especially as concerns how and why employee stock option pools were established and how they change.' (Or something!)

File: README.md [hunk 7]
```diff
@@ -230,28 +230,28 @@ it would be misleading to explain one without the other.
...
 
-- [**Equity Compensation Basics**](#equity-compensation-basics): Here we dive into the fundamentals
-  of compensation and when and whether you can sell equity for cash.
-  These are important concepts to learn in order to understand what equity might be worth.
-- [**How Equity is Granted**](#how-equity-is-granted): This section covers the forms in which equity
-  is most commonly granted, including restricted stock grants, stock options, and RSUs.
+Finally the meat of our discussion: [**How Equity is Granted**](#how-equity-is-granted). This
```
Comment (RacheltheEditor) [inline README.md]: Elide the meat. 🥩 (Consider keeping the structure of the roadmap intact--just the chapter titles and descriptions, no intros. Ditto "We start with" at the top. Not needed!)

Comment (RacheltheEditor) [reply README.md]: (Readers don't expect interruptions before chapter titles in a TOC like this. Well enough without.)

Comment (RacheltheEditor) [reply README.md]: (Also, the roadmap is something we'll want as regular as possible across Guides, so simplifying it structurally will help with expectations imo.)

File: README.md [hunk 8]
```diff
@@ -230,28 +230,28 @@ it would be misleading to explain one without the other.
...
+section covers the forms in which equity is most commonly granted, including restricted
+stock grants, stock options, and RSUs.
 
-Next is taxes:
+Now is where it gets messier—taxes:
```
Comment (RacheltheEditor) [inline README.md]: These breaks make much more sense to me b/c they're separating the Guide out into Parts. I would have us consider even formalizing them, incorporating "Part I," "Part II," "Part III" IIII, V) into the roadmap and even into design within the Guide itself. These kinds of milestones/guideposts can be incredibly comforting, and they build trust in the logic of the structure.

File: README.md [hunk 9]
```diff
@@ -310,10 +311,78 @@ This Guide *complements but does not replace* the advice you get from profession
...
+the form of equity.
 
 In this Guide we focus on equity compensation in stock corporations, the kind of company
 where ownership is represented by stock.
+We describe stock in [the next section](#fundamentals-of-stock-corporations). People are very
+rarely granted stock directly, with no strings attached.
+Instead, they’re given stock with additional restrictions placed on it, or are given
+contractual rights that later can lead to owning stock.
+These forms include restricted stock, stock options, and RSUs, each of which we’ll
+[describe in detail](#how-equity-is-granted).
```
Comment (RacheltheEditor) [inline README.md]: Very pleased with this! Introducing the topics ppl will hope to see and telling them when we'll get to them or how to jump to them, while making sure everyone is actually on the same page w/ what we are talking about when we talk about equity. This structure has both narrative sense and simple-to-complex foundational logic.

File: README.md [hunk 10]
```diff
@@ -440,6 +495,69 @@ Many decisions around granting equity to employees are approved by the board of
 
 🚧 This section could be expanded, and also include more legal links.
 
+### IPOs
```
Comment (RacheltheEditor) [inline README.md]: Public and private companies followed by how private companies might become public companies 👍

File: README.md (hunk 2 above)
Comment (RacheltheEditor) [reply README.md]: 👍 just introducing secondary market here puts what was previously unhoused but important into context that makes it stick!

File: README.md (hunk 4 above)
Comment (jlevy) [reply README.md]: Fixed

File: README.md (hunk 5 above)
Comment (jlevy) [reply README.md]: The reason for this is also the mouseovers. I'll adjust language.

File: README.md (hunk 6 above)
Comment (jlevy) [reply README.md]: Added a variation on this.

File: README.md (hunk 8 above)
Comment (jlevy) [reply README.md]: Yeah. I'd like to revisit that in the future. Remind me if I don't get to it!

**Comment by jlevy** [comment]
@RacheltheEditor I'll merge now I've addressed your comments, but do PR on this if you think anything else needs urgent attention!

File: README.md (hunk 8 above)
Comment (RacheltheEditor) [reply README.md]: 👍

**Pull request merged**